snapcraft/internal/repo/snaps.py

```diff
@@ -61,6 +61,9 @@ def is_valid_snap(cls, snap):
 
     @classmethod
     def is_snap_installed(cls, snap):
+        # Hack for windows
```

Comment:
I have seen some snaps working under WSL.  But generally speaking, I think returning false here is fine here as they aren't supported on Windows (yet?), with or without WSL.

Why are we checking if snaps are installed on Windows..?  I'd like to see the call stack on this one because I am guessing there is something further up the stack we need to change.  

That said, I'd reword the comment ("Hack for Windows" => "Snaps are unsupported on Windows." or something like that

Reply:
Here's the callstack: 
![image](https://user-images.githubusercontent.com/1124174/74967723-69b41780-53ce-11ea-985c-7c055dabd305.png)

Here's the most relevant bit: https://github.com/snapcore/snapcraft/blob/master/snapcraft/internal/project_loader/_config.py#L244
It calls it when it checks to see if the base is installed to determine if it should install it (it also checks if its in a container, which Windows will fail)

Reply:
Gotcha, thanks.

It boils down to snapcraft trying to clean up `prime` in case `snapcraft try` was executed.
https://github.com/snapcore/snapcraft/blob/master/snapcraft/cli/lifecycle.py#L355

This should be harmless.  Though `try` and this cleanup case should probably be disabled on Windows.

Reply:
Ok, I have disabled the clean step for the prime directory on Windows.